src/librustc_codegen_ironox/back/write.rs

```diff
@@ -0,0 +1,11 @@
+#![allow(dead_code)]
+
+macro_rules! asm {
+    ($m:expr, $($args:expr)*) => {
+        $(
+            $m.asm.push_str(&format!("{}\n", $args));
+        )*
+    }
+}
+
+// XXX not yet implemented
```

Comment:
You probably don't want to use `XXX` in rustc, as the tidy linter will reject it. Use `FIXME` instead.

(I much prefer `XXX` too, but we have to play by upstream's rules).

Reply:
There are other instances of XXX which need to be fixed too.